py/repl.c

```diff
@@ -59,12 +59,7 @@ bool mp_repl_continue_with_input(const char *input) {
         || str_startswith_word(input, "class")
         ;
 
-    // check for unmatched open bracket or quote
-    #define Q_NONE (0)
-    #define Q_1_SINGLE (1)
-    #define Q_1_DOUBLE (2)
-    #define Q_3_SINGLE (3)
-    #define Q_3_DOUBLE (4)
```

Comment:
Will put it back! The motivation behind it: they were getting defined every time the mp_repl_continue_with_input was called. Perhaps just put it just below the MICROPY_HELPER_REPL on line 33?